src/main/java/mate/academy/hibernate/relations/exception/DataProcessingException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.hibernate.relations.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message) {
+        super(message);
+    }
+
+    public DataProcessingException(String message, Throwable cause) {
+        super(message);
+    }
```

Comment:
The constructor should pass both 'message' and 'cause' to the superclass. Currently, it only passes the 'message'. Make sure to modify the constructor to also include the cause parameter: super(message, cause);